src/components/topbar/shop/shop.tsx

```diff
@@ -1,10 +1,8 @@
 import { FC, useState, useMemo } from 'react';
```

Comment:
Can we remove/comment some unused variables in this file?
```
src/components/topbar/shop/shop.tsx
  Line 6:8:  '_' is defined but never used         @typescript-eslint/no-unused-vars
  Line 8:8:  'Outgoing' is defined but never used  @typescript-eslint/no-unused-vars
  Line 9:8:  'Claims' is defined but never used    @typescript-eslint/no-unused-vars
```

Reply:
On it now